### ISSUE TITLE
Create a new Contact Us page with up-to-date info

### ISSUE THREAD

**Issue by charleslavon** [near/near-discovery#1064]
The page should contain all of the links below which currently only appear in the near.org footer, additionally it should contain neardevhub.org and [devhub's telegram](https://t.me/NEARDevHub), and perhaps the Subscribe to Newsweek option as well.

<img width="194" alt="Screen Shot 2024-03-13 at 2 45 32 PM" src="https://github.com/near/near-discovery/assets/1915116/95c3d13c-44a0-43ea-a53b-0ac059c65154">

**Comment by charleslavon** [comment]
@vanessadespain wdyt?

**Comment by vanessadespain** [comment]
Totally agree! Thank you for starting this!

**Comment by xuan-ytr** [comment]
See [design intake form here](https://pagodaplatform.atlassian.net/l/cp/CEhuEWMk)

**Comment by xuan-ytr** [comment]
Note to maybe add customer support CTA at lower right --> @reiokam

**Comment by xuan-ytr** [comment]
**[Draft designs](https://www.figma.com/proto/WrsCT1wIeEXd41BOGMBRrS/Contact-Us-Page?page-id=0%3A1&type=design&node-id=3-1188&viewport=-1169%2C-14%2C1.04&t=5F34A0B4iI8LQWbD-1&scaling=min-zoom&mode=design)** ready for review, with open questions. 
[Raw figma here](https://www.figma.com/file/WrsCT1wIeEXd41BOGMBRrS/Contact-Us-Page?type=design&node-id=0%3A1&mode=design&t=n5dQMqzgoQVqZUm4-1).
Since I will be out until 4/10/2024, @vanessadespain could you help unblock @charleslavon and @calebjacob during that time?

**Comment by xuan-ytr** [comment]
cc @reiokam could you weigh in on the right call-to-action / channels to direct and triage requests for support?

This appears in the logged in context. User is most likely a developer, but could be any current user with a near account.

**Comment by vanessadespain** [comment]
I think this page will also be available from the logged-out/marketing experience — replacing [this page](https://pages.near.org/about/contact-us/) that's available from the "About" menu on near.org

I can help keep this effort moving while Xuan is OOO <3

**Comment by reiokam** [comment]
Here's the direct link to the Support Help Center - https://nearhelp.zendesk.com/hc/en-us/requests/new?ticket_form_id=360005876614

Here's the direct link to another Support Form that could be purposed for Developers - https://nearhelp.zendesk.com/hc/en-us/requests/new?ticket_form_id=13521080858775

**Comment by charleslavon** [comment]
@vanessadespain @xuan-ytr Good questions. We're currently using that existing logged-out contact us page on the new logged-in UI as the support link. I'm mostly interested in the redesign for the logged in experience. It can be up to Laura if they want to use this new contact us page for the marketing site.

**Comment by vanessadespain** [comment]
TLDR: I've put together three variations of the support cards from Xuan's design to weigh which support features we utilize.  [View here](https://www.figma.com/proto/WrsCT1wIeEXd41BOGMBRrS/Contact-Us-%2F-Support-Page?page-id=0%3A1&type=design&node-id=29-7015&viewport=-221%2C-531%2C0.16&t=0W4DVgkPPUVLGMJh-1&scaling=min-zoom&starting-point-node-id=29%3A7015&show-proto-sidebar=1&mode=design)
V1 - There are 2 support forms, one for resolving an issue and the other for providing feedback. 
V2 - Link 1 goes to the Zendesk Help Center, Link 2 goes to the support form
V3 - Link 1 opens Zendesk's Ai chat bot, Link 2 goes to the support form. 

Hey @reiokam! After reviewing the support links you shared it looks like they're both the same form, just one has already selected "General Inquiry" from the dropdown (https://nearhelp.zendesk.com/hc/en-us/requests/new?ticket_form_id=360005876614) 

And the other is without selection in the dropdown (https://nearhelp.zendesk.com/hc/en-us/requests/new?ticket_form_id=13521080858775) 

I tracked that the URLs match up when I toggle between those two states. Perhaps we just link to the [form](https://nearhelp.zendesk.com/hc/en-us/requests/new?ticket_form_id=13521080858775) that has no selection made in the dropdown? And/or maybe we can add an option to that dropdown for "Development issues"?

In the event we don't have a chat support feature we could possibly use the "Chat" card in the design to give Developers a direct path to a Developer support form? I'm curious @charleslavon if you think this is a valuable support path for developers — i'm not confident that this is a pattern that developers will use, but we could try and see. 

@reiokam Additional questions for you: 
- Can you confirm that we don't have a customer support chat feature? I saw in Zendesk there's an Ai chat bot. 
- Could you put together a list of Frequently Asked Questions and answers we could use to populate the FAQ?

I also noticed that on the [zendesk page](https://nearhelp.zendesk.com/hc/en-us) the Near logo in the top left goes to the wallet transition page instead of near.org.

**Comment by reiokam** [comment]
Thanks @vanessadespain ! 

Yup, both forms will lead to the same end-point on the Zendesk side. Sorry, after reviewing looks like there were some changes that were made with the recent "Multi-Chain Support" changes, these two were previously similar forms with one form having more options and fields to get more details from the user. Happy to update it to include the "Development Issues" one. 

- Confirmed that we don't have the chat feature right now available and we haven't activated the AI feature. 
- Yup, happy to put some FAQS together maybe 3 or 4 questions, let me know if more is needed? (Thinking Developer Resources, Expectations of Responses?)

**Comment by calebjacob** [comment]
>Yup, happy to put some FAQS together maybe 3 or 4 questions, let me know if more is needed? (Thinking Developer Resources, Expectations of Responses?)

I was going to ask about the FAQs for the technical implementation. Sounds like we'll just be hardcoding some content and don't need to worry about building out a robust FAQs management system - at least for now.

**Comment by vanessadespain** [comment]
@calebjacob Yeah I think for now we'll just hardcore the content and we could consider later on if this will need updating more regularly.

**Comment by reiokam** [comment]
@vanessadespain let me know when you would ideally like the FAQ completed by and I can prioritize on my side? Would Google Docs work, but let me know if you have another preference?

**Comment by vanessadespain** [comment]
@reiokam EOW would be ideal I think so that Engineering can pick it up with next sprint. Is that doable? Ping me if you’d like to collab or see any content in the context of the designs. 

Google doc should be good though! 

With the update you gave I think that the option 2 I shared would be the route to go, relating to the card content. One of the cards leads to you ZenDesk Help Center, allowing more content to be added there without needing to necessarily engage engineering to update this Contact page. And the second card would link to that support form you shared.

**Comment by calebjacob** [comment]
Moving this to `blocked` for the dev work until we've confirmed the designs are ready. What's your status on the design side @vanessadespain? I'm seeing a few versions in the Figma file and I'm not sure what's ready to implement. Thanks!

**Comment by vanessadespain** [comment]
Here is the design direction with the support channels that are most accurate to what's used by the Support team currently. [Shown in Figma here](https://www.figma.com/file/WrsCT1wIeEXd41BOGMBRrS/Contact-Us-%2F-Support-Page?type=design&node-id=29-6225&mode=design)

Status: Copy is still WIP, especially for the [FAQ section](https://docs.google.com/document/d/11fEM-FUBOnEdhNZNfp1hVHWsOVdjqdbRRq8809OQLMc/edit).

![image](https://github.com/near/near-discovery/assets/8346374/7e91dca5-bc8e-478a-9724-02113c4062e4)

**Support Links**
Help Center: https://nearhelp.zendesk.com/hc/en-us
Support Form: https://nearhelp.zendesk.com/hc/en-us/requests/new

**Social Channel Links** 
DevHub: https://near.org/devhub.near/widget/app
Telegram: https://t.me/cryptonear
Discord: https://near.chat
Discourse: https://gov.near.org
Facebook: ???
Reddit: https://www.reddit.com/r/nearprotocol/
Warpcast: ???
YouTube: https://www.youtube.com/channel/UCuKdIYVN8iE3fv8alyk1aMw
GitHub: https://github.com/near
Docs: https://docs.near.org
WeChat: https://pages.near.org/wechat?_gl=1*1bs2mbz*_ga*MjcwMzgxMjUuMTcwOTc1MTM3Mg..*_ga_9GWCXQJ62J*MTcxMjYwMTY0MC42MC4xLjE3MTI2MDIzNTcuMzcuMC4w
X: https://twitter.com/nearprotocol

@xuan-ytr Do you have guidance on a couple of these missing links? And what the counts next to each link should reference? Any additional thoughts/changes you'd like to see?

**Comment by calebjacob** [comment]
Thanks for the update! I have some other tickets I can work on for now while we wait for the copy to be finalized.

**Comment by calebjacob** [comment]
@vanessadespain @xuan-ytr It looks like our Facebook URL would be this: https://www.facebook.com/NEARProtocol

I'm going to hide the Warpcast link until we get a URL - couldn't find that one with Google.

**Comment by xuan-ytr** [comment]
> @xuan-ytr Do you have guidance on a couple of these missing links? 
Thanks @calebjacob for hunting down the Facebook link!
Warpcast: https://warpcast.com/~/channel/near

> And what the counts next to each link should reference? 
We can omit the counts for now since they aren't relevant for every platform. The original thought was that it could generate some excitement to see that we have 1.9M followers on Twitter, for instance. 

> Any additional thoughts/changes you'd like to see?
All looks good to me!
